src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
Sorry; I feel the question is constantly changing; I am arguing not that this fork is bad, but that upstream search/evaluation changes are more likely to have an adverse effect for particular variants (and therefore I'm not uniformly critical of all PRs) as well as attempting & failing to directly answer questions.

While I have concerns about "how in general for any patch to identify whether it is simple" I'll attempt to stick to testable opinions at the risk of sounding inconsistent about when in general I expect such tests would be useful.